tests/test_wave_03.py

```diff
@@ -1,26 +1,66 @@
 import pytest
 
 from adagrams.game import score_word
+score_dict = {  'A' : 1, 'E' : 1, 'I' : 1, 'O' : 1, 'U' : 1, 'L' : 1, 'N' : 1, 'R' : 1, 'S' : 1, 'T' : 1,
+                'D' : 2, 'G' : 2,
+                'B' : 3, 'C' : 3, 'M' : 3, 'P' : 3,
+                'F' : 4, 'H' : 4, 'V' : 4, 'W' : 4, 'Y' : 4,
+                'K' : 5,
+                'J' : 8, 'X' : 8,
+                'Q' : 10, 'Z' : 10}
 
 def test_score_word_accurate():
+    #Arrange
+    WORD_1 = "A"
+    WORD_2 = "DOG"
+    WORD_3 = "WHIMSY"
+    #Act
+    result_1 = score_word(WORD_1)
+    result_2 = score_word(WORD_2)
+    result_3 = score_word(WORD_3)
+
     # Assert
-    assert score_word("A") == 1
-    assert score_word("DOG") == 5
-    assert score_word("WHIMSY") == 17
+    assert result_1 == 1
+    assert result_2 == 5
+    assert result_3 == 17
```

Comment:
As discussed with Jamie during our 1:1, y'all didn't have to break up the tests into Arrange, Act, and Assert. Having the arrange and act portion explicitly written out is optional and a common pattern in writing tests.

However, y'all will see the shorthand version (the way originally wrote this test) in the wild as well.